src/Form/Fields.php

```diff
@@ -173,7 +201,7 @@ public function toArray(Closure|null $map = null): array
 	 */
 	public function toFormValues(): array
 	{
-		return $this->toArray(fn ($field) => $field->toFormValue());
+		return $this->toValues('toFormValue');
```

Comment:
`Suggestion:` I think passing an actual Closure instead of a string name. I think it should work as `$this->toValues($this->toFormValue(...))`.

Allows us more easily to not overlook this e.g. when the method name changes.

Reply:
I have not internalized this new PHP awesomeness yet :)

Reply:
Ah, too bad. It does not seem to work. I guess because of $this not pointing to the field object, but the collection. Or maybe I'm doing something wrong.

Reply:
Ah true, sorry did read it as `$this->toFormValue()` not `$field->toFormValue()`.

I would still suggest to rather keep passing the existing closure to the new `::toValues()` instead just the string. It just feels a bit more reliable than the method name as string.

Reply:
I've just changed it.